### PR TITLE
Update node-cron definitions to match v3

### PR DESCRIPTION
**Pull request by aseidma**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/node-cron/node-cron/blob/v3.0.0/src/scheduled-task.js
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/node-cron/index.d.ts [hunk 1]
```diff
@@ -15,7 +16,6 @@ export function validate(cronExpression: string): boolean;
 export interface ScheduledTask {
     start: () => this;
     stop: () => this;
-    destroy: () => void;
     getStatus: () => string;
```
Comment (maximelkin) [inline types/node-cron/index.d.ts:19]: getStatus missing too https://github.com/node-cron/node-cron/blob/fbc403930ab3165ffef7d53387a29af92670dfea/src/scheduled-task.js#L7

File: types/node-cron/index.d.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+// Type definitions for node-cron 3.0
 // Project: https://github.com/node-cron/node-cron, https://github.com/merencia/node-cron
 // Definitions by: morsic <https://github.com/maximelkin>,
 //                 burtek <https://github.com/burtek>,
 //                 Richard Honor <https://github.com/RMHonor>
 //                 Ata Berk YILMAZ <https://github.com/ataberkylmz>
+//                 Alex Seidmann <https://github.com/aseidma>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import { Timezone } from 'tz-offset';
```
Comment (maximelkin) [inline types/node-cron/index.d.ts:10]: probably replace with just string, not sure if tz-offset matches with moment-timezone

**Comment by maximelkin** [comment]
`schedule()` now also allow to pass pass to the file

File: types/node-cron/index.d.ts (hunk 1 above)
Comment (ataberkylmz) [reply types/node-cron/index.d.ts:19]: @aseidma could you please revise?

**Review by maximelkin — CHANGES_REQUESTED** [review]
(Reviewed code by simple comments, just stub text to satisfy github)

File: types/node-cron/index.d.ts (hunk 1 above)
Comment (aseidma) [reply types/node-cron/index.d.ts:19]: Will revise later today!

**Comment by aseidma** [comment]
Could anyone please explain why the tslint `no-outside-dependancies` rule throws a bunch of error similar like
```
no-outside-dependencies  File /Users/alex/Documents/Code/open-source/DefinitelyTyped/node_modules/@types/node/zlib.d.ts comes from a `node_modules` but is not declared in this type's `package.json`.  See: https://github.com/Microsoft/dtslint/blob/master/docs/no-outside-dependencies.md
```
when developing locally but passes CI without problems? I would really appreciate it!

File: types/node-cron/index.d.ts [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 //                 burtek <https://github.com/burtek>,
 //                 Richard Honor <https://github.com/RMHonor>
 //                 Ata Berk YILMAZ <https://github.com/ataberkylmz>
+//                 Alex Seidmann <https://github.com/aseidma>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
-import { Timezone } from 'tz-offset';
+import EventEmitter = require('events');
 
 export function schedule(cronExpression: string, func: () => void, options?: ScheduleOptions): ScheduledTask;
```
Comment (maximelkin) [inline types/node-cron/index.d.ts:11]: schedule also accepts string instead of func

**Comment by elibarzilay** [comment]
@maximelkin, @burtek, @RMHonor, @ataberkylmz -- ?

**Review by burtek — APPROVED** [review]

**Comment by aseidma** [comment]
Ready to merge

**Pull request merged**